### PR TITLE
Camp thermometer refactor

### LINKED ISSUE #231: Kamp thermometer refactoren
Ik heb laatst een aanpassing gedaan aan de code voor de kamp thermometer (members home page)... ik weet niet wat ik precies had gesnoven toen ik dit schreef, maar het is verschrikkelijk, zowel in de controller als in de blade. Het mag wel een eigen class / vue template / whatever worden. En dat wil ik best zelf doen, maar niet nu en daarom schrijf ik het hier op 😘

### PR DESCRIPTION
**Pull request by robinklaassen**
Closes #231 

Bit of scope creep, also refactored the birthday list for members since it's on same page/controller.

### CODE REVIEW

**Comment by robinklaassen** [comment]
@blackshadev ready for review wat mij betreft, zoals in chat gemeld ben ik niet heel gelukkig met de check op deelnemer betaling, maar ik heb op dit moment nog geen betere ideeën, suggesties welkom :)

File: resources/views/pages/home-member.blade.php [hunk 1]
```diff
@@ -48,55 +18,46 @@
...
+		@if ($birthdays->isNotEmpty())
 			<hr/>
 			<h3>Vandaag jarig:</h3>
 			<ul style="font-size:120%;">
-				@foreach ($today as $member)
-					<li>{{$member->voornaam}} {{$member->tussenvoegsel}} {{$member->achternaam}} ({{$member->leeftijd}}) @if ($member->ikjarig) <strong>&larr; dat ben jij! Gefeliciteerd!</strong> <span class="glyphicon glyphicon-gift"></span> @endif
+				@foreach ($birthdays as $m)
+					<li>
+						{{$m->volnaam}} ({{$m->geboortedatum->age}}) 
+						@if (Auth::user()->profile == $m) <strong>&larr; dat ben jij! Gefeliciteerd!</strong> <span class="glyphicon glyphicon-gift"></span> @endif
```
Comment (blackshadev) [inline resources/views/pages/home-member.blade.php:28]: Can you multiline this (line break after the if, and before the endif) it makes it way  better readable.

File: resources/views/pages/home-member.blade.php [hunk 2]
```diff
@@ -48,55 +18,46 @@
...
-				@endif
-				@if ($c['num_D_bijna'] > 0)
-				<div class="progress-bar progress-bar-warning" role="progressbar" style="width: {{$c['perc_D_bijna']}}%;">{{$c['num_D_bijna']}}</div>
-				@endif
-			</div>
-			<div class="progress-label-right">{{ $c['num_D_goed'] + $c['num_D_bijna'] }} / {{$c['streef_D']}}</div>
-		</div>
+			<camp-thermometer-bar 
+				label="L" 
+				:number-full="{{ $c->members()->wherePivot('wissel', 0)->where('vog', 1)->count() }}"
```
Comment (blackshadev) [inline resources/views/pages/home-member.blade.php:51]: Maybe hide this in a method of the event, not sure.

File: resources/views/pages/home-member.blade.php [hunk 3]
```diff
@@ -48,55 +18,46 @@
...
-				@if ($c['num_D_bijna'] > 0)
-				<div class="progress-bar progress-bar-warning" role="progressbar" style="width: {{$c['perc_D_bijna']}}%;">{{$c['num_D_bijna']}}</div>
-				@endif
-			</div>
-			<div class="progress-label-right">{{ $c['num_D_goed'] + $c['num_D_bijna'] }} / {{$c['streef_D']}}</div>
-		</div>
+			<camp-thermometer-bar 
+				label="L" 
+				:number-full="{{ $c->members()->wherePivot('wissel', 0)->where('vog', 1)->count() }}" 
+				:number-partial="{{ $c->members()->wherePivot('wissel', 0)->where('vog', 0)->count() }}"
```
Comment (blackshadev) [inline resources/views/pages/home-member.blade.php:52]: Maybe hide this in a method of the event, not sure.

File: resources/views/pages/home-member.blade.php [hunk 4]
```diff
@@ -48,55 +18,46 @@
...
-		</div>
+			<camp-thermometer-bar 
+				label="L" 
+				:number-full="{{ $c->members()->wherePivot('wissel', 0)->where('vog', 1)->count() }}" 
+				:number-partial="{{ $c->members()->wherePivot('wissel', 0)->where('vog', 0)->count() }}" 
+				:target-number="{{ $c->streeftal }}">
+			</camp-thermometer-bar>
+			<camp-thermometer-bar 
+				label="D" 
+				:number-full="{{ $c->participants()->wherePivot('datum_betaling', '>', '0000-00-00')->count() }}"
```
Comment (blackshadev) [inline resources/views/pages/home-member.blade.php:57]: Maybe hide this in a method of the event, not sure.

File: resources/views/pages/home-member.blade.php [hunk 5]
```diff
@@ -48,55 +18,46 @@
...
+			<camp-thermometer-bar 
+				label="L" 
+				:number-full="{{ $c->members()->wherePivot('wissel', 0)->where('vog', 1)->count() }}" 
+				:number-partial="{{ $c->members()->wherePivot('wissel', 0)->where('vog', 0)->count() }}" 
+				:target-number="{{ $c->streeftal }}">
+			</camp-thermometer-bar>
+			<camp-thermometer-bar 
+				label="D" 
+				:number-full="{{ $c->participants()->wherePivot('datum_betaling', '>', '0000-00-00')->count() }}" 
+				:number-partial="{{ $c->participants()->wherePivot('datum_betaling', '0000-00-00')->count() }}"
```
Comment (blackshadev) [inline resources/views/pages/home-member.blade.php:58]: Maybe hide this in a method of the event, not sure.

**Review by blackshadev — APPROVED** [review]

**Pull request merged**